dhall/dhall.cabal

```diff
@@ -559,6 +559,7 @@ Library
         scientific                  >= 0.3.0.0  && < 0.4 ,
         template-haskell                           < 2.15,
         text                        >= 0.11.1.0 && < 1.3 ,
+        th-lift-instances           >= 0.1.13   && < 0.2 ,
```

Comment:
Needed to automatically derive `Lift` in various cases. If we don't want to pollute downstream users of `dhall` with orphan instances, I can look into writing these out by hand.

Reply:
I'm not that familiar with `Lift`, but my reasoning is that if these instances are basically unambiguous (i.e. there's only one right way to define them), then I don't think our downstream users will mind

Reply:
Yea, the idea of `Lift` is "given this Haskell value, give me source code that would produce the same value". So `lift [1,2,3] = App (:) 1 (App (:) 2 (App (:) 3 [])))`. I think we can trust these instances to be canonical, but just missing from the respective libraries for whatever reason.